specification/dimensions/chargedescription.md

```diff
@@ -0,0 +1,30 @@
+# Charge Description
+
+A Charge Description provides a high-level context of a line item without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.
+
+The Charge Description column MUST be present within the dataset, MUST be of type String, and SHOULD NOT be null.  Providers SHOULD specify the length of this column in their publicly available FOCUS documentation.
+
+## Column ID
+
+ChargeDescription
+
+## Display Name
+
+Charge Description
+
+## Description
+
+A brief, human-readable summary of a line item.
```

Comment:
Pairing with my other suggestion above...
```suggestion
Self-contained, summary explanation of what the charge is for and how it was priced in a single sentence.
```

Reply:
"Self-contained, summary explanation of what the charge is for and how it was priced".  We have a clause in the description saying that the provider should define the length of this column